srcpkgs/wezterm/template

```diff
@@ -0,0 +1,87 @@
+# Template file for 'wezterm'
+pkgname=wezterm
+version=20220408
+revision=1
+_srcver=20220408-101518-b908e2dd
+wrksrc="${pkgname}-${_srcver}"
+build_style=cargo
+build_helper="rust"
+make_check_args="--workspace --locked --exclude wezterm-ssh"
+hostmakedepends="cargo git pkg-config python3"
```

Comment:
why is git required?

Reply:
It seems it may not be. 🤷

Reply:
I believe it's because some things in the build system like to fallback to git info, for example [here](https://github.com/wez/wezterm/blob/main/config/build.rs#L58)

Reply:
The tarball I selected is based on a tagged release (and includes all Git submodules); so it _should_ have a `.tag` file and thus not lean on answers from Git tools.